tools/istio-iptables/pkg/capture/testdata/ipnets-with-kube-virt-interfaces.golden

```diff
@@ -23,4 +23,4 @@ ip6tables-save
 -I PREROUTING 1 -i eth2 -d 10.0.0.0/8 -j ISTIO_REDIRECT
 -A ISTIO_OUTPUT -d 10.0.0.0/8 -j ISTIO_REDIRECT
```

Comment:
I think this, with dropping the `-p tcp`, ends up being a behavioral change?

IIUC this means any UDP traffic matching this IP range will be redirected which is not going to work

Reply:
Oh I guess `ISTIO_REDIRECT` has `-p tcp` as well...

Reply:
Yeah - this is kinda why I want protocol selectors to be on rules OR chains - but _not_ both at the same time, it just makes it much harder to reason about than it needs to be.

All the redirect rules should have protocol selectors, and do,  AFAICT.